apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/transaction_test.exs

```diff
@@ -1230,6 +1230,24 @@ defmodule OMG.WatcherRPC.Web.Controller.TransactionTest do
                )
     end
 
+    @tag fixtures: [:alice, :more_utxos]
```

Comment:
do you need these fixtures to test this endpoint? it seems like it stateless (as long as watcher is running)?

do you need to have the same validation on the childchain as well?

Reply:
Valid point on the fixtures, I'll try to remove them. Nothing to do on the Childchain side, the `/transaction.create` is a helper endpoint only present in the Watcher. The childchain properly detects merges because it has the list of inputs / ouputs.

Reply:
So it all breaks if I remove the fixtures, unfortunately. I'm gonna qualify the work to figure out why this test fails without fixtures as out of scope for this PR. 

However, since we're not going to merge this right away, I'll try to revisit it once I've been through some high priorities tasks.